src/vc.rs

```diff
@@ -200,6 +200,12 @@ pub enum URI {
     String(String),
 }
 
+impl Default for URI {
+    fn default() -> Self {
+        Self::String("".into())
```

Comment:
The empty string is not a valid URI. I understand this change is so that Delegation and Invocation can be constructed using `Default::default`.

I read in the zcap-ld spec that "An invocation SHOULD have an id"; could `Option<URI>` could be used for `id` in `Invocation`? But for Delegation I read `id` is required: "Every capability document, except for the target, MUST have: an associated id ...".

Could use of `..Default::default()` be replaced with something like `..Delegation::new(id)` with a `new` function?

Reply:
good point, the `::default` was only used in tests anyway